lab-andrew/data/notes/01e96a80-23a1-11e6-a15d-bb1be47ef168

```diff
@@ -0,0 +1 @@
+{"id":"01e96a80-23a1-11e6-a15d-bb1be47ef168","content":"trolls say wat","dateCreated":"2016-05-27T00:21:51.017Z"}
```

Comment:
You should gitignore your 'data' directory, your program should create any necessary folders/files it needs at runtime.  This way when using it locally you have your data folder with persistent information, but when you upload to Github your repository stays reasonably sized and doesn't upload all your data.